lib/formatters/index.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2012 Mark Cavage, Inc.  All rights reserved.
```

Comment:
I assumed that a copyright notice on an open source project was a leftover from a copy and paste. I can add it back. I mean, technically, I wouldn't have the right to make a copy for a fork without his permission.